src/ActionHandler/EnrichOrganizationHandler.php

```diff
@@ -157,7 +159,8 @@ public function run(array $data, array $configuration): array
             $organizationId = $this->data['response']['_self']['id'];
         }//end if
 
-        return $this->service->enrichOrganizationHandler($data, $configuration, $organizationId);
+
+        return $this->service->enrichOrganizationHandler($this->data, $configuration, $organizationId);
```

Comment:
I would use or $this->data & $this->configuration here, or use $data and $configuration